### DISCUSSION TITLE
Disabled world for attached item command

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#5787] [Enhancement]
currently we can attach commands to all items of our choice.
a nice thing..

it would be nice to be able to block some of these items with attached commands for choosen worlds

Option 1: would be a blockAttachedForWorld command....
Option2: a Chest-GUI like we have it for the gamerules - where all worlds are listed and where we have to paste the attached items that are allowed to used. This option might need an extra window with "allWorlds" for items that are allowed in all worlds.

If a world does not contain any allowed attached commands in the GUI then the box will be shows red.

![image](https://user-images.githubusercontent.com/12080017/119608982-23324180-bdf7-11eb-900c-3d7625fbf802.png)

might also need buttons for multiple pages if you have many many items enabled for a world.

**Comment by Zrips** [comment]
Cant you just use specialized commands check ifinworld:[value][@][?][#]! and prevent commands from being performed in specific ones? Or are you just want one thing where you could define where those attached command can only work?

**Comment by CloudeLecaw** [comment]
A thing where you can define all world where those commands attached to an item work or where it doesnt work. Ifinworld is a good option for one world. The ifinworld has the bad site that you need to craeate one for each world where u want it the itemnti work in.

This specialy gets a bit tricky if you have a bit more of these items.

The problem with most checks is that they have no chain abillity for AND and OR.

These would be nice to have in a lot of checks on many places. Not only for ifinworld.

Example: its not possible at the moment to say ifinworld:world|world_nether|farmworld!
Mostly the | is used for OR like known from regex.
